picard/script/__init__.py

```diff
@@ -116,10 +116,11 @@ def get_file_naming_script(settings):
         str: The text of the file naming script if available, otherwise None
     """
     from picard.script import get_file_naming_script_presets
-    if settings["file_renaming_scripts"] and settings["selected_file_naming_script_id"]:
-        selected_id = settings["selected_file_naming_script_id"]
-        if selected_id in settings["file_renaming_scripts"]:
-            return settings["file_renaming_scripts"][selected_id]["script"]
+    scripts = settings["file_renaming_scripts"]
+    selected_id = settings["selected_file_naming_script_id"]
```

Comment:
You can also replace the occurence in ` log.error()` below.

Reply:
Argh!  I completely missed that.  My only excuse is that I just woke up (but maybe not quite it seems).  :blush: